packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -29,83 +29,69 @@ export default {
       control: "text",
       defaultValue: "Starship",
     },
-    label2: {
+    isDisabled: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    onClick: {
       control: "text",
       defaultValue: null,
     },
-    label3: {
+    onDelete: {
       control: "text",
       defaultValue: null,
     },
-    clickable: {
-      control: "boolean",
-      defaultValue: false,
-    },
-    disabled: {
-      control: "boolean",
-      defaultValue: false,
-    },
-    deletable: {
-      control: "boolean",
-      defaultValue: false,
-    },
   },
   decorators: [MuiThemeDecorator],
-};
+} as Meta<TagProps>;
 
-const handleDelete = () => {
-  console.info("You clicked the delete icon.");
+const DefaultTemplate: Story<TagProps> = (args) => {
+  return (
+    <Tag
+      label={args.label}
+      isDisabled={args.isDisabled}
+      onClick={args.onClick}
+      onDelete={args.onDelete}
+    />
+  );
 };
 
-const DefaultTemplate: Story = (args) => {
+const ListTemplate: Story<TagProps> = (args) => {
   return (
-    <Stack direction="row" spacing={2}>
-      <Chip
+    <TagList>
+      <Tag
         label={args.label}
-        clickable={args.clickable}
-        disabled={args.disabled}
-        onDelete={args.deletable && handleDelete}
+        isDisabled={args.isDisabled}
+        onClick={args.onClick}
+        onDelete={args.onDelete}
       />
-      {args.label2 && (
-        <Chip
-          label={args.label2}
-          clickable={args.clickable}
-          disabled={args.disabled}
-          onDelete={args.deletable && handleDelete}
-        />
-      )}
-      {args.label3 && (
-        <Chip
-          label={args.label3}
-          clickable={args.clickable}
-          disabled={args.disabled}
-          onDelete={args.deletable && handleDelete}
-        />
-      )}
-    </Stack>
+      <Tag label="Another tag" />
+      <Tag label="A third tag" />
+    </TagList>
   );
 };
 
 export const Default = DefaultTemplate.bind({});
 Default.args = {};
 
-export const List = DefaultTemplate.bind({});
-List.args = {
-  label2: "Warp-capable",
-  label3: "Unmanned",
-};
+export const List = ListTemplate.bind({});
+List.args = {};
 
 export const Clickable = DefaultTemplate.bind({});
 Clickable.args = {
-  clickable: true,
+  onClick: () => {
+    return true;
+  },
```

Comment:
```suggestion
  onClick: action('onClick'),
```

Described in another post.

You probably know this, but for anyone else, this is only necessary because they're undefined in the default props.